animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -249,6 +259,35 @@ class DistributeMatricesNode(bpy.types.Node, AnimationNode):
 
         return matrices
 
+    def execute_Fibonacci(self, Py_ssize_t amount, Py_ssize_t centermask, float angle, float startsize, float endsize, float scale):
+        cdef Py_ssize_t i 
+
+        if centermask >= amount:
+            centermask = amount
+        i = centermask 
+
+        cdef float factor = 1 / <float>((amount-centermask) - 1) if (amount-centermask) > 1 else 0
```

Comment:
Always use space on both sides of an operation,
```
cdef float factor = 1 / <float>((amount - centermask) - 1) if (amount - centermask) > 1 else 0
```

Reply:
@3DSinghVFX Updated changes